### PR TITLE
Fn to rescue tokens from levered positions

### PR DESCRIPTION
**Pull request by vminkov**
## Description

Fn to rescue tokens from levered positions

could be utilized after a new position is created and used as extension

### CODE REVIEW

File: packages/contracts/contracts/ionic/levered/LeveredPosition.sol [hunk 1]
```diff
@@ -163,6 +165,13 @@ contract LeveredPosition is LeveredPositionStorage, IFlashLoanReceiver {
     }
   }
 
+  function rescueTokens(IERC20Upgradeable asset) external {
+    if (msg.sender != factory.owner()) revert OnlyFactoryOwner();
+    if (asset == stableAsset || asset == collateralAsset) revert AssetNotRescuable();
+
+    asset.transfer(positionOwner, asset.balanceOf(address(this)));
```
Comment (rhlsthrm) [inline packages/contracts/contracts/ionic/levered/LeveredPosition.sol:172]: Should we make sure position is closed first?

Comment (vminkov) [reply packages/contracts/contracts/ionic/levered/LeveredPosition.sol:172]: I wouldn't worry about that, since it's an admin only fn

Comment (rhlsthrm) [reply packages/contracts/contracts/ionic/levered/LeveredPosition.sol:172]: OK sounds good!

File: packages/contracts/contracts/ionic/levered/LeveredPositionFactoryFirstExtension.sol [hunk 2]
```diff
@@ -109,7 +109,7 @@ contract LeveredPositionFactoryFirstExtension is
   }
 
   function getPositionsExtension(bytes4 msgSig) external view returns (address) {
-    // TODO
+    // TODO return the address of some freshly created levered position
```
Comment (rhlsthrm) [inline packages/contracts/contracts/ionic/levered/LeveredPositionFactoryFirstExtension.sol:112]: Does this TODO need to be handled right now?

Comment (vminkov) [reply packages/contracts/contracts/ionic/levered/LeveredPositionFactoryFirstExtension.sol:112]: No, after we deploy and create a new position contract

Comment (9764319764319764) [reply packages/contracts/contracts/ionic/levered/LeveredPositionFactoryFirstExtension.sol:112]: hamed

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**